packages/react-slider/src/components/Slider/Spec.md

```diff
@@ -1,146 +1,42 @@
-| stepToMarks  | x                                                                                       | x                                                                                     | Whether the **Slider** should step to adjacent `marks`. If `true` and the `marks` prop is not provided, then `marks` will be set to `true`.                                  |
-
-### Visual behavior props
-
-| Name     | <img src="https://img.shields.io/badge/Used%20in-v0-orange" alt="drawing" width="200"/> | <img src="https://img.shields.io/badge/Used%20in-v8-blue" alt="drawing" width="200"/> | Description                                                                          |
-| -------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
-| disabled | &check;                                                                                 | &check;                                                                               | Whether to render the **Slider** as disabled. @defaultvalue `false` (render enabled) |
-| vertical | &check;                                                                                 | &check;                                                                               | Whether to render the **Slider** vertically. @default `false` (render horizontally)  |
-| size     | x                                                                                       | x                                                                                     | The size of the Slider.                                                              |
-
-### Event handlers props
-
-| Name     | <img src="https://img.shields.io/badge/Used%20in-v0-orange" alt="drawing" width="200"/> | <img src="https://img.shields.io/badge/Used%20in-v8-blue" alt="drawing" width="200"/> | Description                                                                                        |
-| -------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
-| onChange | &check;                                                                                 | &check;                                                                               | Triggers a callback when the value has been changed. This will be called on every individual step. |
-
-## Migration
```

Comment:
migration moved to Migration.md

Reply:
Seems like that file has not been added to this PR yet.

Reply:
It was already in the repo. Just stating I removed it from here because it was redundant.